test/shimmed.js

```diff
@@ -32,6 +38,11 @@ test('shimmed', function (t) {
 		st.end();
 	});
 
+	t.test('hasOwnProperty', function (st) {
+		st.ok(Object.prototype.hasOwnProperty.call(Symbol, 'for'), 'for is Symbol\' own property');
```

Comment:
Are there other properties that should be checked in this way? Perhaps `keyFor`, and all the well-known symbols as well?